containers/map-api/routes/trigger.js

```diff
@@ -0,0 +1,165 @@
+const express = require("express");
+const router = express.Router();
+
+const Triggers = require("../models/trigger");
+
+// endpoints for beacon
+router.post("/add", function(req, res) {
+  // JSON in req.body
+  // Insert input validation
+  let addTrigger = new Triggers(req.body);
+  addTrigger.save(function(err) {
+    if (err) {
+      res.send(err);
+    } else {
+      res.send("Saved trigger.");
+    }
+  });
+});
+
+router.get("/", function(req, res) {
+  Triggers.find(function(err, triggers) {
+    if (err) {
+      res.send(err);
+    } else {
+      res.send(triggers);
+    }
+  });
+});
+
+router.get("/total", function(req, res) {
+  Triggers.find(function(err, triggers) {
+    if (err) {
+      res.send(err);
+    } else {
+      // res.send(triggers);
+      let jsonData = {
+        zone_one_enter: 0,
+        zone_one_exit: 0,
+        zone_two_enter: 0,
+        zone_two_exit: 0,
+        zone_three_enter: 0,
+        zone_three_exit: 0,
+        zone_four_enter: 0,
+        zone_four_exit: 0,
+        zone_five_enter: 0,
+        zone_five_exit: 0,
+        zone_six_enter: 0,
+        zone_six_exit: 0,
+        zone_seven_enter: 0,
+        zone_seven_exit: 0,
+        zone_eight_enter: 0,
+        zone_eight_exit: 0,
+        zone_nine_enter: 0,
+        zone_nine_exit: 0,
+        zone_ten_enter: 0,
+        zone_ten_exit: 0,
+        zone_eleven_enter: 0,
+        zone_eleven_exit: 0,
+        zone_twelve_enter: 0,
+        zone_twelve_exit: 0,
+        zone_thirteen_enter: 0,
+        zone_thirteen_exit: 0,
+        zone_fourteen_enter: 0,
+        zone_fourteen_exit: 0,
+        zone_fifteen_enter: 0,
+        zone_fifteen_exit: 0
+      };
+      triggers.forEach(function(trigger) {
```

Comment:
For future implementation this function will have to change because it will not scale up. What happens when we have 100 zones 👀